tests/sentry/event_manager/test_event_manager.py

```diff
@@ -354,9 +361,9 @@ def save_event(ts_offset):
         # be able to find it anymore at all if we did not have hierarchical hashes.
 
         # After expiry, new events are still assigned to the same group:
-        # project.update_option("sentry:secondary_grouping_expiry", 0)
-        # event3 = save_event(4)
-        # assert event3.group_id == event2.group_id
+        project.update_option("sentry:secondary_grouping_expiry", 0)
```

Comment:
if this code now works, you can remove the FIXME comment above as well

Reply:
Will follow up